test/parallel/test-dgram-bind-shared-ports.js

```diff
@@ -4,16 +4,21 @@ var assert = require('assert');
 var cluster = require('cluster');
 var dgram = require('dgram');
 
-// TODO XXX FIXME when windows supports clustered dgram ports re-enable this
-// test
-if (process.platform == 'win32')
-  process.exit(0);
-
 function noop() {}
 
 if (cluster.isMaster) {
   var worker1 = cluster.fork();
 
+  if (common.isWindows) {
+    var checkErrType = function(er) {
+      assert.equal(er.code, 'ENOTSUP');
+      worker1.kill();
+    };
+
+    worker1.on('error', common.mustCall(checkErrType, 1));
```

Comment:
1 is the default, you can just write it as `worker1.on('error', common.mustCall(function(err) {` (although it's not wrong this way.)